subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -843,8 +886,9 @@ static int __wifi_args_to_params(const struct shell *sh, size_t argc, char *argv
 		{"eap-pwd6", required_argument, 0, 'P'},
 		{"eap-pwd7", required_argument, 0, 'P'},
 		{"eap-pwd8", required_argument, 0, 'P'},
-		{"ignore-broadcast-ssid", required_argument, 0, 'i'},
+		{"ignore-broadcast-ssid", required_argument, 0, 'g'},
 		{"ieee-80211r", no_argument, 0, 'R'},
+		{"iface", required_argument, 0, 'i'},
```

Comment:
Shouldn't this be optional argument, if user does not give it we figure it out then?